packages/frontend/src/stores/messagelist.ts

```diff
@@ -354,12 +344,22 @@ class MessageListStore extends Store<MessageListState> {
               // See https://github.com/deltachat/deltachat-desktop/issues/4284
               scrollIntoViewArg: { block: 'center' },
             })
-            ActionEmitter.emitAction(
-              chat.archived
-                ? KeybindAction.ChatList_SwitchToArchiveView
-                : KeybindAction.ChatList_SwitchToNormalView
-            )
           })
+
+          // TODO why do we only do this when `firstUnreadMsgId !== null`?
```

Comment:
good question, maybe it was meant as an optimisation attempt. I think we can try to change it.